packages/client/src/i18n/messages/views/advancedSearchBirthForm.ts

```diff
@@ -0,0 +1,99 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
+ * graphic logo are (registered/a) trademark(s) of Plan International.
+ */
+import { defineMessages, MessageDescriptor } from 'react-intl'
+
+interface IAdvancedSearchMessages
+  extends Record<string | number | symbol, MessageDescriptor> {
+  registrationDetails: MessageDescriptor
+  placeOfRegistrationlabel: MessageDescriptor
+  placeOfRegistrationHelperText: MessageDescriptor
+  dateOfRegistration: MessageDescriptor
+  statusOfRecordLabel: MessageDescriptor
+  informantDetails: MessageDescriptor
+  recordStatusAny: MessageDescriptor
+  recordStatusInprogress: MessageDescriptor
+  recordStatusInReview: MessageDescriptor
+  recordStatusRequireUpdate: MessageDescriptor
+  recordStatusRegistered: MessageDescriptor
+  recordStatusCertified: MessageDescriptor
+  recordStatusAchived: MessageDescriptor
+}
+
+const messagesToDefine: IAdvancedSearchMessages = {
+  registrationDetails: {
+    defaultMessage: 'Registration details',
+    description: 'The title of advanced search birth tab form',
+    id: 'form.section.advancedSearch.birth.title'
+  },
+  placeOfRegistrationlabel: {
+    defaultMessage: 'Place of registration',
+    description: 'Label for input Place of registration',
+    id: 'form.field.label.birth.placeOfRegistration'
+  },
+  placeOfRegistrationHelperText: {
+    defaultMessage: 'Search for a province, district or registration office',
+    description: 'Helper text for input Place of registration',
+    id: 'form.field.label.birth.placeOfRegistrationHelperText'
```

Comment:
Let's make the ids for all these messages a bit uniform. Something like `advancedSearch.birth.title`, `advancedSearch.birth.placeOfRegistration`, `advancedSearch.birth.placeOfRegistrationHelperText` etc

Reply:
Okay